admin/billing/payment/stripe_webhook.go

```diff
@@ -0,0 +1,139 @@
+package payment
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/stripe/stripe-go/v79"
+	"github.com/stripe/stripe-go/v79/webhook"
+	"go.uber.org/zap"
+)
+
+// handleWebhook handles incoming webhook events from Stripe
+func (s *Stripe) handleWebhook(w http.ResponseWriter, r *http.Request) {
+	const MaxBodyBytes = int64(65536)
```

Comment:
Why make it exported? Also consider putting consts at the top of the file instead of inside a function.